grow/pods/podcache.py

```diff
@@ -41,12 +55,59 @@ def document_cache(self):
         """Cache for specific document properties."""
         return self._document_cache
 
+    @property
+    def object_cache(self):
+        """Global object cache."""
+        return self.get_object_cache(self.KEY_GLOBAL)
+
+    def create_object_cache(self, key, write_to_file=False, can_reset=True, values=None):
```

Comment:
Makes sense - though instead of using the podcache a user that wants to cache stuff in runtime only could just use werkzeug's cache or some other caching? Or is the idea to centralize caching here and provide a canonical interface?